File: ProjectEuler/Problem_1/sol.cs
using System;

namespace ProjectEular
{
    class sol
    {
        static void Main(string[] args)
        {
            int sum = 0;

            for (int i = 3; i < 1000; i++)
            {
                if (i % 3 == 0)
                {
                    sum += i;
                    // continue;
                }
                else if (i % 5 == 0)
                {
                    sum += i;
                }
            }

            Console.WriteLine (sum);
        }
    }
}

File: ProjectEuler/Problem_2/sol.java

class sol {
    public static void main (String args[]){
        int sum = 0;
        int first = 1;
        int second = 2;

        while (second < 4_000_000){
            if (second % 2 == 0){
                sum += second;
            }

            second = second + first;
            first = second - first;
        }
        System.out.println(sum);
    }
}
